Public/Src/Utilities/Configuration/ILoggingConfiguration.cs

```diff
@@ -197,7 +197,7 @@ public interface ILoggingConfiguration : IWarningHandling
         /// <summary>
         /// When enabled, sends telemetry information for remote collection. Defaults to false.
         /// </summary>
-        RemoteTelemetry RemoteTelemetry { get; }
+        RemoteTelemetry? RemoteTelemetry { get; }

```

Comment:
I'm surprised to see that `/remoteTelemetry-` wasn't already sufficient

Reply:
What's the difference between this property being `null` and `Disabled`?  I'm still not convinced you had to change this type from `RemoteTelemetry` to `RemoteTelemetry?`

Reply:
The test runs with /inCloudBuild. In that mode, remote telemetry is enabled.